inst/tutorials/vaccines/vaccines.Rmd

```diff
@@ -0,0 +1,332 @@
+---
+title: "Vaccines"
+output: learnr::tutorial
+runtime: shiny_prerendered
+---
+
+```{r setup, include=FALSE}
```

Comment:
We want to make the exercises as transparent as possible.
The users can't see any of the code executed in the chuck and so these element appear "by magic" and are a bit black box.
I suggest perhaps simplifying the visualisation (maybe just a prevalence of just an incidence plot?). Then we want ot be stepping the user through how to extract the variables, format and make the plots - it is a really important step for them to learn. You'll then need to include any additional elements e.g. `year = 365` in the code chunks the user sees

Reply:
I think we can drop the # doses parts completely, as Lucy will cover those bits in more details further along

Reply:
It looks to me like the code hidden in the set up chunk has now been made visible in the first section so think this has been addressed. Also looks like the doses section has been removed.

I have also removed the incidence/prevalence plotting function code from the user-facing chunks so we now just have the function calls / plots. I think however, that to help them with the exercises, I will de-function the plotting code and add it to the chunks so they have something to work with/from.